### ISSUE TITLE
Can't add to shopping cart

### ISSUE THREAD

**Issue by HazemKhaled** [Knawat/knawat-dropshipping-magento2#80]
# Test environments Used:
Magento ver. 2.4.1 Enterprise Edition (PHP 7.4.10)
Magento ver. 2.3.6 Enterprise Edition (PHP 7.3.22)
Magento Commerce on Cloud 2.4.0 (PHP 7.4.5)
Magento ver. 2.4.1 Community Edition (PHP 7.4.10)
Magento ver. 2.3.6 Community Edition (PHP 7.3.22)

1.Reasons for failure: Cannot add product to the cart.

Steps:
1. Install the extension
2. On storefront add product to the cart.

AR: "We can't add this item to your shopping cart right now." message appears.

Video: Yes  https://bluejeans.com/s/Siy@8WpKsFg

**Comment by jayjittv** [comment]
@HazemKhaled 
I am able to add products into cart , used above steps and checked with 2.4 version and 2.3.6 version 
I think video shows that there is no knawat keys added and that might be an issue over there 
Please check and confirm 

![image](https://user-images.githubusercontent.com/45030198/104014269-c8e26c80-51d8-11eb-9ac3-8a3fe02befb3.png)

**Comment by HazemKhaled** [comment]
Yes I know, this feedback from Magento market place, they require if the keys not there shouldn't break the website

**Comment by jayjittv** [comment]
> Yes I know, this feedback from Magento market place, they require if the keys not there shouldn't break the website

I will have to check for this by removing key but then one more issue arise , if we allow add to cart without key then what should we do for next operations , like order etc.. 
Its not possible without key

**Comment by jayjittv** [comment]
> > Yes I know, this feedback from Magento market place, they require if the keys not there shouldn't break the website
> 
> I will have to check for this by removing key

I have checked this with given magento version and without knawat key and its working fine , I am able to add product to cart as well placed the order for the same

**Comment by iamdharmesh** [comment]
Thanks for update @jayjittv . @HeshamIsmail this issue looks like store specific can you please provide store details to @jayjittv So, we can check on that store.
